api/src/Entity/Activity.php

```diff
@@ -122,6 +114,15 @@ class Activity extends AbstractContentNodeOwner implements BelongsToCampInterfac
     #[Groups(['read', 'write'])]
     public string $location = '';
 
+    /**
```

Comment:
Why do you move this property down?

Reply:
Automatically done by IDE because I changed the property to private.